src/errors.h

```diff
@@ -3513,5 +3513,7 @@ EXTERN char e_method_str_type_mismatch_expected_str_but_got_str[]
 	INIT(= N_("E1407: Member \"%s\": type mismatch, expected %s but got %s"));
 EXTERN char e_aptypes_is_null_str_nr[]
 	INIT(= "E1408: Internal error: ap_types or ap_types[idx] is NULL: %d: %s");
+EXTERN char e_interface_static_direct_access_str[]
+	INIT(= N_("E1409: Interface static \"%s\": cannot directly access"));
```

Comment:
I didn't like the wording either. So for an interface, `Iface` static variable `s_var1` your suggestion would be
```
E1409: Cannot directly access interface "s_var1" static member
```
That reads a little funny too. How about
```
E1409: Cannot directly access interface static member "s_var1"
```
I thought only naming the variable would be sufficient since there's the line number.
But, assuming the name of the interface is available, it could be changed to
```
E1409: Cannot directly access interface "Iface" static member "s_var1"
```

I won't make a change until hearing back, and there may be more changes to come ;-)

Reply:
There's the question of list_instructions mentioned at https://github.com/vim/vim/pull/13007#issuecomment-1705341446

Something should be done (next PR is fine); suggestions/comments?

Reply:
As this PR is getting bigger, can you create a separate PR for the list_instructions() change?  We also need to add a lot more tests to the test_vim9_disassemble.vim file for Vim9 classes, objects and interfaces.